paper-radio-button.html

```diff
@@ -125,12 +133,17 @@
         position: relative;
         display: inline-block;
         vertical-align: middle;
-        margin-left: 10px;
+        margin-left: var(--paper-radio-button-label-spacing, 10px);
         white-space: normal;
         pointer-events: none;
         color: var(--paper-radio-button-label-color, --primary-text-color);
       }
 
+      :host-context([dir="rtl"]) #radioLabel {
+        margin-left: 0px;
+        margin-right: var(--paper-radio-button-label-spacing, 10px);
```

Comment:
Same answer as before. I'm leaning towards "no"? Not sure. Maybe "no, but wait and see" ?